st2tests/st2tests/api.py

```diff
@@ -180,7 +180,13 @@ class APIControllerWithIncludeAndExcludeFilterTestCase(object):
     # _get_model_instance method method
     test_exact_object_count = True
 
+    # True if those tests are running with rbac enabled
+    rbac_enabled = False
+
     def test_get_all_exclude_attributes_and_include_attributes_are_mutually_exclusive(self):
+        if self.rbac_enabled:
+            self.use_user(self.users['admin'])
```

Comment:
Can you explain where self.rbac_enabled is set to True and also how is adding the user here test the changes?